webui/app/index.html

```diff
@@ -0,0 +1,113 @@
+<html>
+
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type" />
+  <script defer src="https://umami.cloudserver.click/homephoner" data-website-id="333358c7-071a-43c8-980a-294e59cb08b1"></script>
```

Comment:
```suggestion
```
Is this your Telemetry? Better to remove in this instance, you can have on your private website tho